code/modules/mob/living/carbon/human/species/species_attack.dm

```diff
@@ -5,11 +5,9 @@
 	sharp = 1
 	edge = 1
 
-/datum/unarmed_attack/diona
-	attack_verb = list("lashed", "bludgeoned")
-	attack_noun = list("tendril")
-	eye_attack_text = "a tendril"
-	eye_attack_text_victim = "a tendril"
+
```

Comment:
In future, I'd recommend something like:
//DS13 - Moved to our modular folder.
Or just a straight up override, deletion like this shouldn't be necessary

Reply:
i was just tidying a few things up here